lib/matplotlib/tests/test_datetime.py

```diff
@@ -8,12 +8,7 @@
 
 
 class TestDatetimePlotting:
-    @pytest.mark.xfail(reason="Test for acorr not written yet")
-    @mpl.style.context("default")
-    def test_acorr(self):
-        fig, ax = plt.subplots()
-        ax.acorr(...)
-
+    @pytest.mark.xfail(reason="Test for annotate not written yet")
```

Comment:
```suggestion
```

The conflict was that this had been removed because that method was implemented, it should not be xfailed again